models/Profile/Event.js

```diff
@@ -34,6 +36,14 @@ const EventSchema = new Schema({
   price: {
     startingFrom: Number,
   },
+  userStatus: {
+    type: String,
```

Comment:
Why is this a string and not a boolean?
There are only 2 states:
attending & not speaking
attending & speaking

This is the perfect use case for a boolean

Reply:
Could there be more? What about making it an object with ...?

- `isSpeaking`
- `isAttending`

Reply:
I was under the assumption that people were only posting events they are attending. That would make isAttending always true. Is this not the case?

Reply:
Yeah I think people are posting events they are aware of or want to support - which I think is a good thing, I am keen to have a central place for events